core/evm/sputnik/sputnik.go

```diff
@@ -0,0 +1,303 @@
+				acc := GetAccount(cache, require.Address())
+				fmt.Println("BAL:",acc.Balance())
+				vm.CommitAccount(require.Address(), new(big.Int).SetUint64(acc.Sequence()), new(big.Int).SetUint64(acc.Balance()), acc.Code())
+			}
+
+		case sputnikvm.RequireAccountCode:
+			fmt.Println("\n RequireAccountCode:", require.Address().Bytes())
+			acc := GetAccount(cache, require.Address())
+			vm.CommitAccountCode(require.Address(), acc.Code())
+
+		case sputnikvm.RequireAccountStorage:
+			fmt.Println("\n RequireAccountStorage:", require.Address().Bytes())
+			converted, addr := fromEthAddress(require.Address(), true)
+			if !converted {
+				fmt.Println("Commit None Exist")
+				vm.CommitNonexist(require.Address())
+				break
+			}
+			key := binary.Uint64ToWord256(require.StorageKey().Uint64())
+			fmt.Println("Address: ", addr)
+			fmt.Printf("Storage key: %d\n", require.StorageKey().Uint64())
+			storage, err := cache.GetStorage(addr, key)
+			if err != nil {
+				vm.CommitAccountStorage(require.Address(), require.StorageKey(), new(big.Int).SetUint64(0))
+				break
+			}
+			var value big.Int
+			value.SetUint64(binary.Uint64FromWord256(storage))
+			fmt.Printf("Storage value: %d\n", binary.Uint64FromWord256(storage))
+			vm.CommitAccountStorage(require.Address(), require.StorageKey(), &value)
+
+		case sputnikvm.RequireBlockhash:
+			fmt.Println("\n RequireBlockhash:", require.Address().Bytes())
+			blockNumber := new(big.Int).SetUint64(bc.LastBlockHeight())
+			var hash common.Hash
+			hash.SetBytes(bc.LastBlockHash())
+			vm.CommitBlockhash(blockNumber, hash)
+
+		default:
+			fmt.Println("\n default...")
+			//panic("Panic : unreachable!")
+			return nil, retError
+		}
+	}
+
+	fmt.Println("\n\nVM Successfuly : ", !vm.Failed())
+	fmt.Printf("VM Output Len : %d\n", vm.OutLen())
+	fmt.Printf("VM Output : %s\n\n", hex.EncodeToString(vm.Output()))
+
+	if vm.Failed() {
+		fmt.Println("\n\nOh! Failed :(")
+		return []byte{}, retError
+	}
+
+	changedaccs := vm.AccountChanges()
+	lacc := len(changedaccs)
+
+	for i := 0; i < lacc; i++ {
+		acc1 := changedaccs[i]
+
+		if acc1.Address().IsEmpty() {
+			continue
+		}
+
+		switch acc1.Typ() {
+
+		case sputnikvm.AccountChangeIncreaseBalance:
+			fmt.Println("AccountChangeIncreaseBalance")
+			amount := acc1.ChangedAmount()
+			fmt.Println("Amount:", amount)
+			fmt.Println("Address:", acc1.Address())
+			targetAcc := GetAccount(cache, acc1.Address())
+			fmt.Println("Address:", targetAcc.Address())
+			targetAcc.AddToBalance(amount.Uint64())
+			fmt.Println("Added Successfully!")
+			cache.UpdateAccount(targetAcc)
+
+		case sputnikvm.AccountChangeDecreaseBalance:
+			fmt.Println("AccountChangeDecreaseBalance")
+			amount := acc1.ChangedAmount()
+			targetAcc := GetAccount(cache, acc1.Address())
+			targetAcc.SubtractFromBalance(amount.Uint64())
+			cache.UpdateAccount(targetAcc)
+
+		case sputnikvm.AccountChangeRemoved:
+			fmt.Println("AccountChangeRemoved")
+			//removeAccount(acc1.Address())
+
```

Comment:
Please add this comment here:
// TODO: We need to support removing account in state...